src/test/java/uk/gov/hmcts/reform/sscs/service/venue/VenueRpcDetailsServiceTest.java

```diff
@@ -0,0 +1,138 @@
+package uk.gov.hmcts.reform.sscs.service.venue;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.sscs.model.VenueDetails;
+import uk.gov.hmcts.reform.sscs.model.VenueDetails.VenueDetailsBuilder;
+import uk.gov.hmcts.reform.sscs.model.venue.VenueRpcDetails;
+import uk.gov.hmcts.reform.sscs.service.VenueDataLoader;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+
+@ExtendWith(MockitoExtension.class)
+class VenueRpcDetailsServiceTest {
+
+    public static final String VALID_EPIMS_ID = "560788";
+    public static final String INVALID_EPIMS_ID = "abcdef";
+    public static final String VENUE_ID = "60";
+
+    private VenueDetailsBuilder venueDetailsBuilder;
+
+    @Mock
+    private VenueDataLoader venueDataLoader;
+
+    @InjectMocks
+    VenueRpcDetailsService venueRpcDetailsService;
+    private Map<String, VenueDetails> venueDetailsMap;
```

Comment:
minor: add a new line above this please - `venueDetailsMap` and `venueDetailsBuilder` should also be grouped together, preferably under the unitUnderTest/venueRpcDetailsService